games-strategy/wesnoth/wesnoth-1.16.10.ebuild

```diff
@@ -44,7 +44,8 @@ BDEPEND="
 "
 
 PATCHES=(
-	"${FILESDIR}"/${PN}-1.16.8-gcc-13.patch
+#	"${FILESDIR}"/${PN}-1.16.8-gcc-13.patch
```

Comment:
Is there a reason to comment this? Probably fine to just remove the old patch line.

Reply:
I first commented it out when it failed to apply but before I found that the patch had been integrated upstream. There is no need for it, I just missed it on final review.